.github/workflows/quick-merge.yml

```
@@ -0,0 +1,170 @@
+      - name: Merge pull request
+        id: merge
+        uses: actions/github-script@f28e40c7f34bde8b3046d885e986cb6290c5673b
+        env:
+          PR_NUMBER: ${{ steps.pr.outputs.pr_number }}
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+            const prNumber = parseInt(process.env.PR_NUMBER, 10);
+            const owner = context.repo.owner;
+            const repo = context.repo.repo;
+
+            const { data: pr } = await github.rest.pulls.get({
```

actionlint (1.7.8)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)